### PR TITLE
Point out token propagation issue

### PR DESCRIPTION
**Pull request by daniel-ac-martin**

### CODE REVIEW

File: how-to-docs/kubernetes-user-token.md [hunk 1]
```diff
@@ -22,6 +22,8 @@
 
 4. Once the token is created the requester should be able to see it in their **Connected Identities** section for use in their Kube config.
 
+**Note:** Tokens can take a few minutes to propagate so you may been to wait a while before using a new token.
```
Comment (jits) [inline how-to-docs/kubernetes-user-token.md]: `so you may been to wait` -> `so you may have to wait for up to 5 mins`

Comment (KashifSaadat) [inline how-to-docs/kubernetes-user-token.md]: -> `so you may have to wait`

**Comment by KashifSaadat** [comment]
Cheers good spot!

The Platform Hub will automatically sync tokens for the clusters (to S3) every 5 minutes, and then it can take a few minutes for the master nodes to reload the tokens file. After 10 minutes max you should be good to go.

**Comment by daniel-ac-martin** [comment]
Fixed?

**Pull request merged**